Tests/JavaKitMacroTests/JavaClassMacroTests.swift

```diff
@@ -78,6 +78,11 @@ class JavaKitMacroTests: XCTestCase {
             public init(javaHolder: JavaObjectHolder) {
                 self.javaHolder = javaHolder
             }
+
+            /// Casting to ``JavaObject`` will never be nil because ``HelloWorld`` extends it.
+            public func `as`(_: JavaObject.type) -> JavaObject {
```

Comment:
Almost, in Swift this has to be `.Type` (uppercase).

Once that's fixed tests will pass as well so I'll take that edit